### DISCUSSION TITLE
Resize engraving area

### DISCUSSION THREAD

**Discussion by Adyknight** [arkypita/LaserGRBL#1934] [General]
Hi all

I am new to CNC so please be kind lol.
I am trying to engrave a small slate coaster 100mm x 100mm.
But I can not find the option to resize my area.
I do not have option 3.
I think iam just needing day but I need your help 😉
Machine I am using is the CNC 3018 pro.
Stay safe.
![sizeandoption~2](https://user-images.githubusercontent.com/14598849/194843021-b64c7621-d08b-49fc-8454-dbbed5ad09e1.jpg)

**Comment by StuartB4** [comment]
You don't get that option if your file is a svg vector file, only png, jpg, bmp etc.

**Reply by Adyknight** [reply]
Ah I see.
So I guess I resize it in inkscape save it as the size I want,then open it in lasergrbl.
Thanks for your help
